app/controllers/landing_page_controller.rb

```diff
@@ -221,8 +221,6 @@ def render_landing_page(default_locale:, locale_param:, structure:)
                          true)
     marketplace_context = marketplace_context(c, topbar_locale, request)
 
-    FeatureFlagHelper.init(request, false, false)
```

Comment:
I guess the original purpose for `false, false` was to render only the community specific feature flags. I'm not sure if that's right thing to do, but good to acknowledge that this change changes what we had before.